src/main/java/com/jfrog/ide/idea/log/Logger.java

```diff
@@ -104,4 +107,9 @@ private static String prependPrefix(String message, NotificationType notificatio
         }
         return "[INFO] " + message;
     }
+
+    public static void openSettings(String details, Project project, Class<? extends Configurable> configurable) {
```

Comment:
openSettings --> addOpenSettingsLink

Let's also add a clear usage description to the method. We can also add this as an example - 
```
Logger.openSettings("It looks like Gradle home was not properly set in your project. Click <a href=\"#settings\">here</a> to set Gradle home.", project, GradleConfigurable.class);
```